RELEASE_NOTES.md

```diff
@@ -1,5 +1,6 @@
 # Upcoming
 
+- feat: Support for parsing UTF8 characters in code and comments if their UTF16 value is on maximum 2 bytes (https://github.com/dafny-lang/dafny/pull/2717)
```

Comment:
```suggestion
- feat: Support for parsing Basic Multilingual Plane characters from UTF-8 in code and comments (https://github.com/dafny-lang/dafny/pull/2717)
```

Nit picking again, I know, but "UTF-8 character" is a bit confusing, since UTF-8 is a way to encode characters using sequences of code units (bytes).